en/resources/contributing.md

```diff
@@ -374,3 +378,145 @@ We are currently working on a new version of the security model, the most update
 If you have suggestions on how this process could be improved please submit a
 pull request.
 
+# Contributing to Expressjs.com {#expressjs-website-contributing}
```

Comment:
i want to add a ‘—-‘ line here to seperate this section.

Reply:
If it's fine, it's fine by me.